### DISCUSSION TITLE
go2rtc shows no URLs and the Cards return error cannot get URL - Am I missing a step?

### DISCUSSION THREAD

**Discussion by protomota** [fuatakgun/eufy_security#1046] [Q&A]
I think I am missing a step or something, but I don't see anything in the documentation related to this. Should the Eufy Security Integration automatically add stream entity IDs and URLs to go2rtc, or do you need to do that manually? I am just not understanding how that part of the integration is supposed to work, and it doesn't currently work for me. Once I setup the Eufy Security Integration, I would expect to see my Eufy Camera Entities listed in the go2rtc streams page, but maybe not. When I add the card to the HA dashboard it returns the error "can't get URL for camera" which I believe is returned from go2rtc trying to find the missing entity IDs and URLs.
<img width="417" alt="Screenshot 2023-12-22 at 8 42 21 AM" src="https://github.com/fuatakgun/eufy_security/assets/1140777/130df7bc-02e0-40e3-9f66-35d8b49d4256">
<img width="516" alt="Screenshot 2023-12-22 at 8 43 08 AM" src="https://github.com/fuatakgun/eufy_security/assets/1140777/5afe46a1-576f-4179-89a4-8b1d5378f401">

**Comment by fuatakgun** [comment]
If you have webrtc, which comes bundled with go2rtc, integration will add cameras to go2rtc automatically when streaming is started

**Comment by InstigatorX** [comment]
Not working me either.
